cves/CVE-2013-2134.yml

```diff
@@ -45,7 +45,9 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  By using wildcard matching (using a generic case that will match if a specific case is not provided),
```

Comment:
Sentences should still be readable when text in () is removed, this actually reads as "By using wildcard matching, an attacker can execute Java code that is not in the source code." I'd recommend rewording as its not clear exactly what the vulnerability is from this.

Reply:
To answer your question, wildcard matching is the latter.